BookApiTest/TestBookServicce.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using Xunit;
+using WebApiTraining.Model;
+using System.Collections.Generic;
+using FluentAssertions;
+
+namespace BookApiTest
+{
+    public class TestBookServicce
+    {
+        [Fact]
+        public void Test_Get_Books()
```

Comment:
Also Define the output , while naming the test method. eg - TestGetBooksReturnsListOfBooks()